src/ng/directive/ngOptions.js

```diff
@@ -591,7 +591,7 @@ var ngOptionsDirective = ['$compile', '$document', '$parse', function($compile,
           element.label = option.label;
           element.textContent = option.label;
         }
-        if (option.value !== element.value) element.value = option.selectValue;
+        element.value = option.selectValue;
```

Comment:
@gkalpak I noticed the same thing.  That's how I new this check would be safe to remove.  My guess is it was intended to be `option.selectValue !== element.value` and the thought was that would prevent setting `element.value` if it didn't need to be.  Strange thing about that though is that this function is only ever called from the `addOptionElement` function and it creates a new element cloned from `optionTemplate` everytime . So, I don't think `element.value` will ever actually have a value prior to this line anyway.  Makes me wonder if the check a few lines above `option.label !== element.label` is necessary.